engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -164,7 +164,9 @@ - (BOOL)handleOpenURL:(NSURL*)url
                                  completionHandler:^(BOOL success) {
                                    if (!success && throwBack) {
                                      // throw it back to iOS
-                                     [UIApplication.sharedApplication openURL:url];
+                                     [UIApplication.sharedApplication openURL:url
+                                                                      options:@{}
+                                                            completionHandler:nil];
```

Comment:
cc @hannah-hyj i may need your help testing this out on a physical device. Do you remember what you did before to trigger this line?

Reply:
yes, to trigger this line, you should set up AASA file for deep link in certain domain, and not support that deep link in your flutter app, so the link will be opened again in browser.

Reply:
i will help testing if this is working